ui/packages/admin-core-logviewer/src/main/webapp/js/filter.js

```diff
@@ -40,7 +40,9 @@ export default (filters, logs) => {
   const regexps = fields.reduce((o, field) => {
     try {
       o[field] = new RegExp(filters[field], 'i')
-    } catch (e) {}
+    } catch (e) {
+      // do nothing
```

Comment:
any chance we want to put something to the console?

Reply:
we could. idk how often this block is hit during completely normal operation though